schema/mongodb/cadence/schema.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "create": "cluster_config"
+  },
+  {
+    "createIndexes": "cluster_config",
+    "indexes": [
+      {
+        "key": {
+          "rowtype": 1,
+          "version": -1
+        },
+        "name": "rowtype_version",
+        "unique": true
+      }
+    ],
+    "writeConcern": {
+      "w": "majority"
+    }
+  }
+]
```

Comment:
do we need some CLI or make commands to install the schema? or maybe that will be in a later diff?

Reply:
yeah, there will be another refactoring to create a `cadence-nosql-tool`. It's in the plan but I want to finish this mongo DB before doing that.